ddtrace/opentrace/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .span import Span
+from .settings import Config
+from .tracer import Tracer
+
+__all__ = [
+    'Config',
+    'Span',
```

Comment:
I think we should "expose" only the `Config` and the `Tracer`. The public API doesn't expose directly spans or scopes. Indeed, everything should pass by `Tracer` API.